src/components/Select.jsx

```diff
@@ -355,18 +353,13 @@ export default class Select extends Component {
       const menuNode = React.findDOMNode(this.refs.menu);
 
       // the menu was just opened
-      if (!previousState.isOpen && this.state.isOpen) {
+      if (!previousState.isOpen && this.state.isOpen && this.props.children && this.props.children.length > 0) {
         const positionOptions = has(this.props, 'positionOptions') ? this.props.positionOptions : config.positionOptions;
         positionOptions(this);
       // restore the old scrollTop position
       } else {
         menuNode.scrollTop = this.cachedMenuScrollTop;
       }
-
-      if (!previousState.isOpen && this.state.isOpen) {
```

Comment:
How it worked before is that the menu is hidden until it's repositioned and then shown again. If you take the out it can lead to a jumping box on slow computers. Better to keep it.